### PR TITLE
Configure jest caching location

### PR DESCRIPTION
**Pull request by zephraph**
This changes the location of the jest cache to ensure that it's cleared between CI builds. 

It also updates failing snapshots from master.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -227,6 +227,7 @@
     "bracketSpacing": true
   },
   "jest": {
+    "cacheDirectory": ".cache/jest",
```
Comment (damassi) [inline package.json]: 👍

**Comment by damassi** [comment]
ty! 

merge on green

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit ed1b470efa6e9111ffd6c3c8b321a0c305944ba8

https://deploy-preview-2524--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit efff417ba70b918d13891751a43a8062727b759c

https://deploy-preview-2524--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
:rocket: PR was released in v16.14.9 :rocket:

**Pull request merged**